.storybook/main.js

```diff
@@ -1,10 +1,12 @@
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  stories: ['../src/client/components/*.stories.{js,md,mdx}'],
```

Comment:
Went really specific to avoid feeling like I needed to alter too much of your code without really knowing what it is supposed to do.